### PR TITLE
Vova_Krychkivskyi_Task#2

### PR DESCRIPTION
**Pull request by VovaKrychkivskyi**
Add task#2 (links.map & socialMediaIcons.map)

### CODE REVIEW

File: src/components/app/App.js [hunk 1]
```diff
@@ -1,24 +1,59 @@
...
+
+            console.log(odd);
+              return <Card post={item} key={item.id} hasImage={odd} />
+          })
+        }
+      </div>
+
+      <div className="d-flex posts-container">
+      {/*  todo: срендерить тут список пользователей, используя компонент UserCard */}
+          {usersList.map((value) => {
```
Comment (AlyonaPianykh) [inline src/components/app/App.js]: please format code properly use ctrl + alt+ shift + L  to format automatically

File: src/components/footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+export function Footer(props) {
+  return (
+    <div className="may-footer">
+      <div className="may-footer-content">
+      <div className="may-footer-text">Some text about authors</div>
+      <div className="may-footer-social-media-wrapper">
+          {socialMediaIcons.map((value, index) => {
+              return(
+                  <a key={index} href={value.src}>{value.alt}</a>
```
Comment (AlyonaPianykh) [inline src/components/footer/Footer.js]: Why did you change tag `image` to tag `a`? it should be kept as is

File: src/components/header/HeaderFromLecture.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  console.log(links);
+  return (
+    <div className="may-header navbar">
+      <img src={Logo} className="may-header-logo" />
+
+      <div className="may-header-links-wrapper">
+          {
+              links.map((value, index) => {
+                  return(
+                      <a key={index} href={value.url}>{value.name}</a>
```
Comment (AlyonaPianykh) [inline src/components/header/HeaderFromLecture.js:19]: good

**Comment by AlyonaPianykh** [comment]
Review done
Please check 1 question

**Comment by VovaKrychkivskyi** [comment]
1/please format code properly use ctrl + alt+ shift + L  to format automatically/-done
2/Why did you change tag image to tag a? it should be kept as is/- I returned tag image
3/Please check 1 question/- done
